README.md

```diff
@@ -1,50 +1,76 @@
+- Changes to the core architecture
+- Introduction of new dependencies
+- Major UI/UX changes
+- Changes affecting backward compatibility
+- Significant performance optimizations
 
-   1. When you are finished, click "Commit Changes" at the top of the page. Enter the title and description from the last step into the appropriate fields. Then, click "Propose changes." This will create a new Pull Request (PR) for your proposal.
+### When an RFC is _Encouraged_
 
-   1. Congratulations! You have successfully submitted an RFC. Now, you can engage in the discussion about your proposal. You can view your PR by navigating to the pull requests tab here: [https://github.com/meshtastic/rfcs/pulls](https://github.com/meshtastic/rfcs/pulls)
+An RFC is encouraged for any initiative where someone or a group of someones wishes to benefit from the collective wisdom and experience of the Meshtastic community, but are **not subject to the rules below**.
 
-1. **Submit Your RFC as a PR**: Create your RFC using the template and submit it as a Pull Request (PR) to this repository. Use the `Summary` section of your RFC as the PR description. Your PR will serve as the primary platform for discussion about your proposal.
+### When an RFC is Not Required
 
-   > **Note:** If you followed the steps to edit your proposal in the GitHub web interface, you can skip this step! Your proposal is already submitted as a PR.
+Minor changes that do not broadly impact the project's functionality or user experience may not require an RFC. This includes:
 
-1. **Participate in the Discussion**: Engage with the community and project maintainers in the PR discussion. This is where your proposal will be reviewed, and feedback will be provided.
+- Bug fixes
+- Minor feature changes that do not alter key functionality
+- Minor UI/UX adjustments that do not alter the overall design
+- Documentation updates
 
-1. **Finalizing the RFC**: The discussion period will remain open until the RFC is deemed complete. Upon completion, the PR will be closed. Note that closing the PR does not necessarily mean the RFC is approved; it signifies the end of the discussion phase.
+## Timeline
 
-## Community Participation
+The typical timeline for an RFC process, which is a minimum of 3 weeks, is as follows:
 
-We highly encourage community involvement in the RFC process. Whether you're proposing an RFC or contributing to the discussion of someone else's, your input is valuable.
+1. **Proposal Submission**: The RFC is submitted to the repository and made available for public comment.
+1. **Discussion Period**: A minimum of 2 weeks is allocated for community discussion and feedback. This period may be extended based on the complexity of the RFC or the volume of feedback received.
```

Comment:
2 weeks felt right to me given availability, vacations, etc. What do y'all think?